webpack.config.js

```diff
@@ -187,6 +187,7 @@ const webpackConfig = {
 		'entry-main': [ path.join( __dirname, 'client', 'boot', 'app' ) ],
 		'entry-domains-landing': [ path.join( __dirname, 'client', 'landing', 'domains' ) ],
 		'entry-login': [ path.join( __dirname, 'client', 'landing', 'login' ) ],
+		'entry-gutenbearding': [ path.join( __dirname, 'client', 'landing', 'gutenbearding' ) ],
```

Comment:
Should we start to move new entry points to `/apps`?

Reply:
Entrypoint is not standalone enough to be an app. It's still part of Calypso's webpack build, shares chunks with other entrypoints, and is served by the same express.js server. The main difference is that it does a full navigation (as opposed to a SPA one using history API) and loads its own HTML document from server.

The Jetpack dashboard by @tyxla will be a good app once we distribute more parts of Calypso framework as packages.